Misc/NEWS.d/next/Library/2024-07-03-07-25-21.gh-issue-121332.Iz6FEq.rst

```diff
@@ -0,0 +1,3 @@
+Fix constructor of :mod:`ast` nodes with custom ``_attributes``. Previously,
```

Comment:
Previously? Isn't it still raising DeprecationWarning?

Reply:
It still raises a DeprecationWarning if you pass attributes the class doesn't know about. But now it no longer raises a warning if you set `_attributes` on a custom subclass and pass those attributes.